Zend/zend_vm_def.h

```diff
@@ -4160,7 +4178,9 @@ ZEND_VM_C_LABEL(fcall_by_name_end):
 		zend_rethrow_exception(execute_data);
 		HANDLE_EXCEPTION();
 	}
-	ZEND_VM_SET_OPCODE(opline + 1);
+	// todo: under what circumstances do ZEND_USER_FUNCTIONs take this code path?
```

Comment:
For `ZEND_DO_FCALL_BY_NAME`, I think this isn't ever taken by user functions. All the expansions of `ZEND_VM_ENTER_EX` seem to return except for when `HYBRID_NEXT` is used, where it does a jump. I'm not sure where the jump goes. Is this dead code for user code? @dstogov